### PR TITLE
test: remove last enzyme tests

### PR DESCRIPTION
**Pull request by jakemhiller**
### Overview

Removes final enzyme tests and libraries from the repo

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [ABC-123]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.9.7-alpha.bedaf1.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.bedaf1.0<br/>@codecademy/gamut-kit@0.6.262-alpha.bedaf1.0<br/>@codecademy/gamut-labs@34.0.9-alpha.bedaf1.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.bedaf1.0<br/>@codecademy/gamut-styles@14.2.5-alpha.bedaf1.0<br/>@codecademy/gamut-tests@4.0.22-alpha.bedaf1.0<br/>@codecademy/gamut@51.7.2-alpha.bedaf1.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.bedaf1.0<br/>@codecademy/monaco@0.6.119-alpha.bedaf1.0<br/>@codecademy/styleguide@62.8.3-alpha.bedaf1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63b897db9e5d250057531942--gamut-preview.netlify.app>https://63b897db9e5d250057531942--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63b897db9e5d250057531942>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.9.7-alpha.24963a.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.24963a.0<br/>@codecademy/gamut-kit@0.6.262-alpha.24963a.0<br/>@codecademy/gamut-labs@34.0.9-alpha.24963a.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.24963a.0<br/>@codecademy/gamut-styles@14.2.5-alpha.24963a.0<br/>@codecademy/gamut-tests@4.0.22-alpha.24963a.0<br/>@codecademy/gamut@51.7.2-alpha.24963a.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.24963a.0<br/>@codecademy/monaco@0.6.119-alpha.24963a.0<br/>@codecademy/styleguide@62.8.3-alpha.24963a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63b89becec314a009498684e--gamut-preview.netlify.app>https://63b89becec314a009498684e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63b89becec314a009498684e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.0.3-alpha.7d0c0b.0<br/>@codecademy/gamut-icons@7.9.7-alpha.7d0c0b.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.7d0c0b.0<br/>@codecademy/gamut-kit@0.6.262-alpha.7d0c0b.0<br/>@codecademy/gamut-labs@34.0.9-alpha.7d0c0b.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.7d0c0b.0<br/>@codecademy/gamut-styles@14.2.5-alpha.7d0c0b.0<br/>@codecademy/gamut-tests@4.0.22-alpha.7d0c0b.0<br/>@codecademy/gamut@51.7.2-alpha.7d0c0b.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.7d0c0b.0<br/>@codecademy/monaco@0.6.119-alpha.7d0c0b.0<br/>@codecademy/styleguide@62.8.3-alpha.7d0c0b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63b8ce1a0e674d26235459f4--gamut-preview.netlify.app>https://63b8ce1a0e674d26235459f4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63b8ce1a0e674d26235459f4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.0.3-alpha.6b6c40.0<br/>@codecademy/gamut-icons@7.9.7-alpha.6b6c40.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.6b6c40.0<br/>@codecademy/gamut-kit@0.6.262-alpha.6b6c40.0<br/>@codecademy/gamut-labs@34.0.9-alpha.6b6c40.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.6b6c40.0<br/>@codecademy/gamut-styles@14.2.5-alpha.6b6c40.0<br/>@codecademy/gamut-tests@4.0.22-alpha.6b6c40.0<br/>@codecademy/gamut@51.7.2-alpha.6b6c40.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.6b6c40.0<br/>@codecademy/monaco@0.6.119-alpha.6b6c40.0<br/>@codecademy/styleguide@62.8.3-alpha.6b6c40.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63b997700e674d33f6545b2d--gamut-preview.netlify.app>https://63b997700e674d33f6545b2d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63b997700e674d33f6545b2d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.0.3-alpha.4aa3ec.0<br/>@codecademy/gamut-icons@7.9.7-alpha.4aa3ec.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.4aa3ec.0<br/>@codecademy/gamut-kit@0.6.262-alpha.4aa3ec.0<br/>@codecademy/gamut-labs@34.0.9-alpha.4aa3ec.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.4aa3ec.0<br/>@codecademy/gamut-styles@14.2.5-alpha.4aa3ec.0<br/>@codecademy/gamut-tests@4.0.22-alpha.4aa3ec.0<br/>@codecademy/gamut@51.7.2-alpha.4aa3ec.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.4aa3ec.0<br/>@codecademy/monaco@0.6.119-alpha.4aa3ec.0<br/>@codecademy/styleguide@62.8.3-alpha.4aa3ec.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63bc59c41f34dc3af2224a89--gamut-preview.netlify.app>https://63bc59c41f34dc3af2224a89--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63bc59c41f34dc3af2224a89>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>eslint-plugin-gamut@2.0.3-alpha.54f1d7.0<br/>@codecademy/gamut-icons@7.9.7-alpha.54f1d7.0<br/>@codecademy/gamut-illustrations@0.32.9-alpha.54f1d7.0<br/>@codecademy/gamut-kit@0.6.263-alpha.54f1d7.0<br/>@codecademy/gamut-labs@34.0.10-alpha.54f1d7.0<br/>@codecademy/gamut-patterns@0.8.22-alpha.54f1d7.0<br/>@codecademy/gamut-styles@14.2.5-alpha.54f1d7.0<br/>@codecademy/gamut-tests@4.0.22-alpha.54f1d7.0<br/>@codecademy/gamut@51.7.2-alpha.54f1d7.0<br/>@codecademy/markdown-overrides@0.9.19-alpha.54f1d7.0<br/>@codecademy/monaco@0.6.119-alpha.54f1d7.0<br/>@codecademy/styleguide@62.8.4-alpha.54f1d7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://63bc69ef745e5c0a70dcc96d--gamut-preview.netlify.app>https://63bc69ef745e5c0a70dcc96d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/63bc69ef745e5c0a70dcc96d>Deploy Logs</a></p>

File: packages/gamut-styles/src/__tests__/GamutProvider.test.tsx [hunk 1]
```diff
@@ -11,65 +12,81 @@ jest.mock('../cache', () => {
...
       children: <GamutProvider theme={theme} />,
     });
 
+    const allStyles = Array.from(
+      document.querySelectorAll('style')
+    ).filter((el) => Boolean(el.getAttribute('data-emotion')));
+
     expect(createEmotionCache).toHaveBeenCalledTimes(1);
-    expect(wrapper.find(Global).length).toEqual(4);
+    expect(allStyles.length).toBeGreaterThan(0);
```
Comment (jakemhiller) [inline packages/gamut-styles/src/__tests__/GamutProvider.test.tsx:72]: these tests were already pretty flimsy, and i'd say these changes probably make them slightly worse. I don't think a lot of these tests are super high-value so we may want to come back and remove some of the less important ones.

**Review by dreamwasp — APPROVED** [review]
lgtm!

**Review by BandanaKM — APPROVED** [review]

**Pull request merged**